qe_metrics/utils/issue_utils.py

```diff
@@ -45,17 +47,20 @@ def mark_obsolete_issues(
         if db_issue.issue_key in closed_issue_keys and db_issue.status != OBSOLETE_STR:
             LOGGER.info(f'Marking issue "{db_issue.issue_key}" for product {product.name} as {OBSOLETE_STR}')
             db_issue.status = OBSOLETE_STR
-    orm.commit()
+    db_session.commit()
 
 
-def update_existing_issue(existing_issue: JiraIssuesEntity, new_issue_data: Issue, severity: str) -> None:
+def update_existing_issue(
+    existing_issue: JiraIssuesEntity, new_issue_data: Issue, severity: str, db_session: Session
+) -> None:
```

Comment:
`new_issue_data` is missing from docstring
`issue` should be removed from docstring